modules/core/src/main/kotlin/datamaintain/core/config/DatamaintainConfig.kt

```diff
@@ -102,7 +102,7 @@ enum class CoreConfigKey(override val key: String,
 
 fun Properties.getProperty(configKey: ConfigKey): String =
         if (configKey.default != null) {
-            getProperty(configKey, configKey.default!!)
+            getProperty(configKey.key, configKey.default!!)
```

Comment:
This ```getProperty``` calls the method defined at line 119 that calls getProperty with the configKey key. 
Either you stop calling it and remove that method, either you keep calling it.